.github/workflows/promote.yaml

```diff
@@ -43,8 +43,8 @@ jobs:
         with:
           credentials: ${{ secrets.CHARMCRAFT_AUTH }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          destination-channel: latest/${{ env.promote-to }}
-          origin-channel: latest/${{ env.promote-from }}
-          charmcraft-channel: latest/stable
+          destination-channel: 1/${{ env.promote-to }}
+          origin-channel: 1/${{ env.promote-from }}
+          charmcraft-channel: 1/stable
```

Comment:
I don't think this one should have changed to `1`, that's the `charmcraft` snap right?

Reply:
oof. Yes, good catch.

Reply:
https://github.com/canonical/self-signed-certificates-operator/pull/305/commits/6a07e66ed26ef1abf47859b01f562d1ba51dc3bb